### PR TITLE
Preserve data

### PR DESCRIPTION
**Pull request by RWUBAKWANAYO**
# In this project we:

- [ ] Saved all the data (books, people, and rentals) into JSON files. One per type of data you are storing and named accordingly (e.g. `rentals.json`)
- [ ]  loaded all data from files on startup of the app. 
- [ ] Handle the case when one or more of the files are missing.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -4,14 +4,10 @@
...
-
 class App
   def initialize
-    @people = []
-    @books = []
-    @rentals = []
-    @list = List.new(@books, @people, @rentals)
-    @create = Create.new(@books, @people, @rentals)
+    @list = List.new
+    @create = Create.new
```
Comment (brytebee) [inline app.rb:10]: - [ ] **Required:** I have noticed that the `main.rb` file hasn't changed in this branch and so I cannot comment on it.

The app crashes when I attempt to create a `Person` (Student and Teacher) or `Book`. Kindly trace the errors from the gif video attached below.

crashing_app

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @RWUBAKWANAYO & @kamzzy👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following: 
- Nicely structured documentation.✔️
- Clean Ruby code and style guide.✔️
- No linter errors.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RWUBAKWANAYO** [comment]
After tracing the error from the application we found that there's a missing `storage` folder that must hold our stored JSON files.
All files inside the `storage` folder can be removed except the folder itself.

**Review by Davidosky007 — APPROVED** [review]
Hi @RWUBAKWANAYO @kamzzy 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written readme file 👍 
* Proper use of Gitflow 👍 
* Data stored properly 👏 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RWUBAKWANAYO** [comment]
merge changes to the dev branch

**Pull request merged**